iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ShuntCompensator.java

```diff
@@ -17,71 +17,79 @@
 public interface ShuntCompensator extends Injection<ShuntCompensator> {
 
     /**
-     * Get the maximum section count.
+     * Get the current count of sections in service.
+     * Please note sections can only be sequentially in service i.e. all sections from section 1 to section currentSectionCount are in service.
+     * <p>
+     * It is expected to be greater than one and lesser than or equal to the
```

Comment:
Why? It should be possible to have 0 section activated?

Reply:
By default for me when 0 section is activated, it means that there is zero susceptance and admittance (cf. above)

Reply:
doc should be updated?